x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1083,6 +1083,11 @@ describe('when on the endpoint list page', () => {
         ).toBe('Policy Response');
       });
 
+      it('should display timestamp', async () => {
+        const timestamp = await renderResult.findByTestId('endpointDetailsPolicyResponseTimestamp');
```

Comment:
nitpick: you can make this code sync and simpler if you use `queryByTestId` instead of `findByTestId`
```
const timestamp =  renderResult.queryByTestId('endpointDetailsPolicyResponseTimestamp');
expect(timestamp).not.toBeNull();
```

in this particular case it doesn't change anything, but I've seen longer tests that get complicated (or confusing) because of the async apis.